spec/requests/api/members_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe "api/members" do
+
+  def json
+    JSON.parse(response.body)
+  end
+
+  describe 'POST api/members' do
+    let(:new_member) { build :member, email: "newbie@test.org" }
+
+    it 'creates a new member' do
+      expect { post api_members_path(:email => new_member.email) }.to change {Member.count}.by(1)
```

Comment:
For the post path, you don't need to pass email. Instead the email should become part of the post body:

```ruby
expect{
  post api_members_path, email: 'test@example.com'
}.to change { Member.count }.by(1)
```